docs/guides/update-guide/connectors/010-to-020.md

```diff
@@ -159,10 +159,10 @@ If using the
 [pre-packaged runtime environment](/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md#pre-packaged-runtime-environment)
 that comes with the SDK does not fit your use case, you can create a custom runtime environment.
 
-In version 0.2.0 of the SDK, the names of the following classes need to be adjusted:
+In version 0.2.0 of the [job worker connector run-time](https://github.com/camunda/connector-sdk/tree/main/runtime-job-worker), the following changes are required:
 
-- Rename `io.camunda.connector.runtime.jobworker.ConnectorConfig` to `io.camunda.connector.runtime.jobworker.outbound.OutboundConnectorConfig`
-- Rename `io.camunda.connector.runtime.jobworker.ConnectorJobHandler` to `io.camunda.connector.runtime.jobworker.outbound.ConnectorJobHandler`
+- Rename `io.camunda.connector.runtime.jobworker.ConnectorJobHandler` to `io.camunda.connector.runtime.jobworker.api.outbound.ConnectorJobHandler`
+- Removed `io.camunda.connector.runtime.jobworker.ConnectorConfig` (unused)
```

Comment:
I think we can simply remove this line altogether, we never suggested using this class anyways, it was just used in internally.